src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Models/ItemTemplateSelectorTestPageViewModel.cs

```diff
@@ -0,0 +1,101 @@
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Uno.UI.Samples.UITests.Helpers;
+using Windows.UI.Core;
+
+namespace Uno.UI.Samples.Presentation.SamplePages
+{
+	public partial class ItemTemplateSelectorTestPageViewModel : ViewModelBase
+	{
+
+		public ItemTemplateSelectorTestPageViewModel(CoreDispatcher coreDispatcher) : base(coreDispatcher)
+		{
+			ListItemsObsStatic = GetSampleItemsSync();
+		}
+
+		public object ListItemsObsStatic { get; }
+
+
+		private ListItem[] GetSampleItemsSync()
+		{
+			return ColourList;
+		}
+
+		public class ListItem
```

Comment:
Codacy found an issue: [This class overrides 'GetHashCode' and should therefore also override 'Equals'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5930374)